source/fab/steps/compile_fortran.py

```diff
@@ -346,6 +346,42 @@ def compile_file(self, analysed_file, flags, output_fpath):
             value={'time_taken': timer.taken, 'start': timer.start})
 
 
+def get_fortran_preprocessor():
+    fpp, fpp_flags = None, None
+
+    try:
+        fpp, fpp_flags = get_tool(os.getenv('FPP'))
+        logger.info(f'env defined fpp as "{fpp}"')
```

Comment:
Shorthand is fine in debug messages but for "Info" level messages I think we prefer whole words and recognised contractions with apostrophes.